packages/atomic/src/utils/result-utils.ts

```diff
@@ -15,27 +14,17 @@ export function bindLogDocumentOpenOnResult(
   resultElement: Element,
   selector?: string
 ) {
-  const logDocumentOpenOnce = once(() => {
-    engine.dispatch(ResultAnalyticsActions.logDocumentOpen(result));
+  const resultLinkAnalytics = buildResultLinkAnalytics(engine, {
```

Comment:
I'd like to hear other's opinions on the name of this controller, and if there are better suggestions. Some ideas:
- buildResult
- buildResultLink
- buildResultTracker?
- buildResultLogger?
- buildResultMonitor?
- buildResultManager?

Reply:
If we decide to include Result/ResultLink/ResultLinkAnalytics objects such as this one with every result in the result list controller, then I'd vote for something like `buildResultLink`. I'd have a slight preference for `buildResult` normally, but since there's already a `Result` interface this could create a mess.

Otherwise, if it stays a bit more of a explicitly declared helper, then it depends on whether we expect it to do more than analytics in the future.

Reply:
I vote `buildResultLink`

Reply:
> it depends on whether we expect it to do more than analytics in the future.

I was thinking last night that this controller could help with all the interactions associated with a displayed result.

Beyond selecting a result, it could power the "must include" feature. It could expose properties like the "highlightedTitle" and "highlightedExcerpt". It could potentially power the quickview feature.

I think this would be a valuable combination to have in one place, versus splitting the functionality over multiple controllers. This would suggest the name should be more general.

Reply:
Alright, go for `buildResultLink` then !

Reply:
Fantastic! I'll rename it `buildResultLink` and rename `delay` to `selectionDelay`.